test/scripts/run_multiple_kibana_nodes.sh

```diff
@@ -0,0 +1,85 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#!/bin/bash
+
+#
+# Script to run multiple kibana nodes in parallel on the same machine.
+# Make sure to run the script from kibana root directory.
+#
+# bash test/scripts/run_multiple_kibana_nodes.sh <function> [options]
+# functions:
+#   start [instances] [args] - start multiple kibanas (3 default)
+#   es [args] - run elasticsearch
+#   tail - show logs of all kibanas
+#   kill - kills all started kibana processes
+#   clean - clean up nohup files
+#   kibana_index - search .kibana index against es
+#
+
+FN="$1"
```

Comment:
optional: Not everyone knows bash as well as JS. What functionality not supported by JS?

Reply:
I don't think there's anything here that couldn't be implemented in JS, but I suspect it would just be a bit harder and more code to deal with async and streaming output between processes/files.

Reply:
Discussed offline, it would be good to have a long term goal of including multi-node upgrade migrations in automated tests. It would be much easier to implement such a test (and remove this script) once we have node clustering support https://github.com/elastic/kibana/issues/68626